<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReaderInfo</class>
 <widget class="QMainWindow" name="ReaderInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1880</width>
    <height>1048</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>个人信息修改</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/res/0005.png</normaloff>:/res/0005.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget{
background:url(:/res/LW (2).jpg);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>1581</width>
      <height>961</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
background:rgba(255,255,157,0.90);
border-radius:15px;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>10</y>
       <width>311</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
color:#00A388;
font-size:50px;
font-family:Microsoft JhengHei;
background:transparent;
}
</string>
     </property>
     <property name="text">
      <string>个人信息修改</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>100</y>
       <width>201</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
color:#00A388;
font-size:25px;
font-family:Microsoft JhengHei;
background:transparent;
}</string>
     </property>
     <property name="text">
      <string>用户名</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>140</y>
       <width>611</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
background:transparent;
border:none;
color:#717072;
font-size:25px;
font-family:Microsoft JhengHei;
border-bottom:1px solid #717072;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>100</y>
       <width>401</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
color:#00A388;
font-size:20px;
font-family:Microsoft JhengHei;
background:transparent;
}</string>
     </property>
     <property name="text">
      <string>15位以内的字母，数字组合</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>210</y>
       <width>201</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
color:#00A388;
font-size:25px;
font-family:Microsoft JhengHei;
background:transparent;
}</string>
     </property>
     <property name="text">
      <string>密码</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>250</y>
       <width>611</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
background:transparent;
border:none;
color:#717072;
font-size:25px;
font-family:Microsoft JhengHei;
border-bottom:1px solid #717072;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
     <property name="placeholderText">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>210</y>
       <width>401</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
color:#00A388;
font-size:20px;
font-family:Microsoft JhengHei;
background:transparent;
}</string>
     </property>
     <property name="text">
      <string>15位以内的字母，数字组合</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>320</y>
       <width>201</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
color:#00A388;
font-size:25px;
font-family:Microsoft JhengHei;
background:transparent;
}</string>
     </property>
     <property name="text">
      <string>姓名</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>360</y>
       <width>611</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
background:transparent;
border:none;
color:#717072;
font-size:25px;
font-family:Microsoft JhengHei;
border-bottom:1px solid #717072;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Name</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_8">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>540</y>
       <width>671</width>
       <height>241</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>201</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
color:#00A388;
font-size:25px;
font-family:Microsoft JhengHei;
background:transparent;
}</string>
      </property>
      <property name="text">
       <string>身份</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_3">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>80</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QRadioButton::indicator {
    width: 40px;
    height: 40px;
}
QRadioButton::indicator::unchecked {
    image:url(:/res/LW_6.png);
}
QRadioButton::indicator::checked {
    image:url(:/res/LW_5.png);
}

QRadioButton{
color:#FF6138;
font-size:28px;
font-family:Microsoft JhengHei;
}</string>
      </property>
      <property name="text">
       <string>大二 Sophomore</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_4">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>130</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QRadioButton::indicator {
    width: 40px;
    height: 40px;
}
QRadioButton::indicator::unchecked {
    image:url(:/res/LW_6.png);
}
QRadioButton::indicator::checked {
    image:url(:/res/LW_5.png);
}

QRadioButton{
color:#FF6138;
font-size:28px;
font-family:Microsoft JhengHei;
}</string>
      </property>
      <property name="text">
       <string>大三 Junior</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_5">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>180</y>
        <width>261</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QRadioButton::indicator {
    width: 40px;
    height: 40px;
}
QRadioButton::indicator::unchecked {
    image:url(:/res/LW_6.png);
}
QRadioButton::indicator::checked {
    image:url(:/res/LW_5.png);
}

QRadioButton{
color:#FF6138;
font-size:28px;
font-family:Microsoft JhengHei;
}</string>
      </property>
      <property name="text">
       <string>教师     Teacher</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_6">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>180</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QRadioButton::indicator {
    width: 40px;
    height: 40px;
}
QRadioButton::indicator::unchecked {
    image:url(:/res/LW_6.png);
}
QRadioButton::indicator::checked {
    image:url(:/res/LW_5.png);
}

QRadioButton{
color:#FF6138;
font-size:28px;
font-family:Microsoft JhengHei;
}</string>
      </property>
      <property name="text">
       <string>大四 Senior</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_7">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>80</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QRadioButton::indicator {
    width: 40px;
    height: 40px;
}
QRadioButton::indicator::unchecked {
    image:url(:/res/LW_6.png);
}
QRadioButton::indicator::checked {
    image:url(:/res/LW_5.png);
}

QRadioButton{
color:#FF6138;
font-size:28px;
font-family:Microsoft JhengHei;
}</string>
      </property>
      <property name="text">
       <string>博士     Doctor</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_8">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>30</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QRadioButton::indicator {
    width: 40px;
    height: 40px;
}
QRadioButton::indicator::unchecked {
    image:url(:/res/LW_6.png);
}
QRadioButton::indicator::checked {
    image:url(:/res/LW_5.png);
}

QRadioButton{
color:#FF6138;
font-size:28px;
font-family:Microsoft JhengHei;
}</string>
      </property>
      <property name="text">
       <string>研究生 Graduate</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_9">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QRadioButton::indicator {
    width: 40px;
    height: 40px;
}
QRadioButton::indicator::unchecked {
    image:url(:/res/LW_6.png);
}
QRadioButton::indicator::checked {
    image:url(:/res/LW_5.png);
}

QRadioButton{
color:#FF6138;
font-size:28px;
font-family:Microsoft JhengHei;
}</string>
      </property>
      <property name="text">
       <string>大一 Freshman</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>820</y>
       <width>201</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
color:#00A388;
font-size:25px;
font-family:Microsoft JhengHei;
background:transparent;
}</string>
     </property>
     <property name="text">
      <string>手机号</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>850</y>
       <width>611</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
background:transparent;
border:none;
color:#717072;
font-size:25px;
font-family:Microsoft JhengHei;
border-bottom:1px solid #717072;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Phone Number</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_9">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>90</y>
       <width>671</width>
       <height>351</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_10">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>800</y>
       <width>671</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_11">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>90</y>
       <width>661</width>
       <height>141</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>50</y>
        <width>301</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox {
background:rgba(255,255,157,0.80);
color:#FF6138;
font-size:28px;
font-family:Microsoft JhengHei;
}
QComboBox:enabled:hover, QComboBox:enabled:focus {
        color: rgb(250, 128, 664);
		background:rgba(255,255,157,0.80);
}
</string>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>计算机学院</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/res/lw (1).png</normaloff>:/res/lw (1).png</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>理学院</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/res/lw (2).png</normaloff>:/res/lw (2).png</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>文法学院</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/res/lw (3).png</normaloff>:/res/lw (3).png</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>生命科学学院</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/res/lw (4).png</normaloff>:/res/lw (4).png</iconset>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_26">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>201</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
color:#00A388;
font-size:25px;
font-family:Microsoft JhengHei;
background:transparent;
}</string>
      </property>
      <property name="text">
       <string>学院</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_12">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>260</y>
       <width>661</width>
       <height>381</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label_27">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>201</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
color:#00A388;
font-size:25px;
font-family:Microsoft JhengHei;
background:transparent;
}</string>
      </property>
      <property name="text">
       <string>问题</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_28">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>201</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
color:#00A388;
font-size:25px;
font-family:Microsoft JhengHei;
background:transparent;
}</string>
      </property>
      <property name="text">
       <string>答案</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_29">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>240</y>
        <width>401</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
color:#00A388;
font-size:20px;
font-family:Microsoft JhengHei;
background:transparent;
}</string>
      </property>
      <property name="text">
       <string>15位以内的中文，字母，数字组合</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_30">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>120</y>
        <width>401</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
color:#00A388;
font-size:20px;
font-family:Microsoft JhengHei;
background:transparent;
}</string>
      </property>
      <property name="text">
       <string>15位以内的中文，字母，数字组合</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>611</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
background:transparent;
border:none;
color:#717072;
font-size:25px;
font-family:Microsoft JhengHei;
border-bottom:1px solid #717072;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Security Question</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_31">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>201</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
color:#00A388;
font-size:25px;
font-family:Microsoft JhengHei;
background:transparent;
}</string>
      </property>
      <property name="text">
       <string>密保问题设置</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>280</y>
        <width>611</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
background:transparent;
border:none;
color:#717072;
font-size:25px;
font-family:Microsoft JhengHei;
border-bottom:1px solid #717072;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Answer</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>1370</x>
       <y>720</y>
       <width>121</width>
       <height>121</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background:#ff8c69;
border-radius:60px;
font-size:28px;
font-family:Microsoft JhengHei;
}
QPushButton:hover{
border-radius:60px;
background:#FF6F69;
font-size:28px;
font-family:Microsoft JhengHei;
}
</string>
     </property>
     <property name="text">
      <string>返回</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>1210</x>
       <y>720</y>
       <width>121</width>
       <height>121</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background:#96CA2D;
border-radius:60px;
font-size:28px;
font-family:Microsoft JhengHei;
}
QPushButton:hover{
background:#168039;
border-radius:60px;
font-size:28px;
font-family:Microsoft JhengHei;
}
</string>
     </property>
     <property name="text">
      <string>提交</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_32">
     <property name="geometry">
      <rect>
       <x>1090</x>
       <y>640</y>
       <width>611</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
color:#00A388;
font-size:20px;
font-family:Microsoft JhengHei;
background:transparent;
}</string>
     </property>
     <property name="text">
      <string>提示：用户名作为您的身份唯一标识，不支持修改</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>450</y>
       <width>671</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label_34">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>201</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
color:#00A388;
font-size:25px;
font-family:Microsoft JhengHei;
background:transparent;
}</string>
      </property>
      <property name="text">
       <string>性别</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QRadioButton::indicator {
    width: 40px;
    height: 40px;
}
QRadioButton::indicator::unchecked {
    image:url(:/res/LW_6.png);
}
QRadioButton::indicator::checked {
    image:url(:/res/LW_5.png);
}

QRadioButton{
color:#FF6138;
font-size:28px;
font-family:Microsoft JhengHei;
}</string>
      </property>
      <property name="text">
       <string>男 Male</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>30</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QRadioButton::indicator {
    width: 40px;
    height: 40px;
}
QRadioButton::indicator::unchecked {
    image:url(:/res/LW_6.png);
}
QRadioButton::indicator::checked {
    image:url(:/res/LW_5.png);
}

QRadioButton{
color:#FF6138;
font-size:28px;
font-family:Microsoft JhengHei;
}</string>
      </property>
      <property name="text">
       <string>女 Female</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <zorder>groupBox_10</zorder>
    <zorder>groupBox_9</zorder>
    <zorder>label_17</zorder>
    <zorder>label_18</zorder>
    <zorder>lineEdit</zorder>
    <zorder>label_19</zorder>
    <zorder>label_20</zorder>
    <zorder>lineEdit_2</zorder>
    <zorder>label_21</zorder>
    <zorder>label_22</zorder>
    <zorder>lineEdit_3</zorder>
    <zorder>groupBox_8</zorder>
    <zorder>label_25</zorder>
    <zorder>lineEdit_4</zorder>
    <zorder>groupBox_11</zorder>
    <zorder>groupBox_12</zorder>
    <zorder>pushButton</zorder>
    <zorder>pushButton_2</zorder>
    <zorder>label_32</zorder>
    <zorder>groupBox</zorder>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
